component/authenticator/src/main/java/org/wso2/carbon/identity/application/authenticator/totp/TOTPAuthenticator.java

```diff
@@ -721,4 +724,14 @@ private void setUserClaimValues(AuthenticatedUser authenticatedUser, String user
 			throw new AuthenticationFailedException(errorMessage, e);
 		}
 	}
+
+	private String buildAbsoluteURL(String redirectUrl) throws URISyntaxException {
```

Comment:
Shall we check the possibility to use the `buildAbsoluteURL` in [1]?

[1] https://github.com/wso2-extensions/identity-outbound-auth-totp/blob/38b34404d0e9834318a3091e3ae2bbf7e7a3c945/component/authenticator/src/main/java/org/wso2/carbon/identity/application/authenticator/totp/TOTPAuthenticator.java#L304